modules/API/API.js

```diff
@@ -164,6 +165,14 @@ function initCommands() {
 
             APP.store.dispatch(toggleTileView());
         },
+        'toggle-raise-hand': () => {
+            sendAnalytics(createApiEvent('raise-hand.toggled'));
+            APP.store.dispatch(participantUpdated({
+                id: APP.conference.getMyUserId(),
+                local: true,
+                raisedHand: !APP.conference._room.getLocalParticipantProperty('raisedHand')
```

Comment:
You can get the current flag state by getting the local participant object instead. This will make this resilient to changes in APP.conference, which is legacy code.